### PR TITLE
Add file url support to doesUrlPointToIndexJson()

### PR DESCRIPTION
**Pull request by lentheivendra**
Signed-off-by: Len Theivendra <theivend@ca.ibm.com>

### CODE REVIEW

**Comment by lentheivendra** [comment]
For #595

**Comment by lentheivendra** [comment]
@tobespc Please review + merge this one too, and let us know if other areas of Portal code need updating.

cc @tetchel @jingfuwang @elsony

File: src/pfe/portal/modules/Templates.js [hunk 1]
```diff
@@ -454,17 +454,33 @@ function isRepo(obj) {
...
 
-  const options = {
-    host: url.host,
-    path: url.pathname,
-    method: 'GET',
+  let templateSummariesText = '[]';
+  if ( url.protocol === 'file:' ) {
+    try {
+      if ( fs.existsSync(url.pathname) ) {
+        let data = fs.readFileSync(url.pathname, "utf-8");
```
Comment (hhellyer) [inline src/pfe/portal/modules/Templates.js]: Could this be changed to `await fs.readFile(url.pathname, "utf-8")` so it doesn't block? (Also fs.existsSync can be changed.)

Comment (lentheivendra) [reply src/pfe/portal/modules/Templates.js]: @hhellyer thanks, changed to await on async fs calls.

Comment (hhellyer) [reply src/pfe/portal/modules/Templates.js]: @lentheivendra - Thanks, I think we had a linter rule that warned about using sync methods. I've made a note to check whether we're running the linter on Jenkins now.

**Comment by hhellyer** [comment]
@lentheivendra - The template repository tests are failing for this PR: https://ci.eclipse.org/codewind/job/Codewind/job/codewind/job/PR-624/5/console
If you need any help getting those setup and running (it doesn't take long) just message me.

**Comment by lentheivendra** [comment]
> @lentheivendra - Their PR: https://ci.eclipse.org/codewind/job/Codewind/job/codewind/job/PR-624/5/console I'm not sure if they are related to this change or not.
> If you need any help getting those setup and running (it doesn't take long) just message me.

This is being tracked via issue #635

**Pull request merged**